README.md

```diff
@@ -0,0 +1,168 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Countries Index <a name="about-project"></a>
+
+**Countries Index** is a web application that allows users to search for countries and get information about them. The information includes the country's name, capital, population, area, and flag and list of timezones. You can also filter the countries by continent.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User can filter countries by continent**
+- **User can see all countries on the home page**
+- **User can see a country's details when they click on it**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://shasherazi.github.io/countriesIndex/)
+
```

Comment:
- [ ] Please make sure that the online version is up and has a working link in your readme.

![image](https://user-images.githubusercontent.com/62290545/225333119-e6fc2b1f-5807-49d4-916b-992d2cc84629.png)